data/dirt belt jobs.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Drought Relief"
+	name `Drought relief to <planet>`
+	job
+	repeat
+	description `The farming world of <destination> is currently experiencing a drought. Deliver <cargo> to the planet by <date> for <payment>.`
+	cargo "drought relief" 25 2 .05
```

Comment:
drought relief -> drought relief supplies

Or maybe "water"?

Reply:
well it could be medication, food, or any other number of things that happen due to a lack of water